### ISSUE TITLE
Build fase not working

### ISSUE THREAD

**Issue by pampeano** [nicklockwood/SwiftFormat#252]
Since version 0.33.6, the build phase not working.

<img width="747" alt="captura de pantalla 2018-04-21 a la s 13 30 21" src="https://user-images.githubusercontent.com/35750464/39086509-cde135d8-4568-11e8-81a4-9fe63907d5c7.png">

My build phase script is

```bash
if which swiftformat >/dev/null; then
  swiftformat "${SRCROOT}" --disable unusedArguments --commas inline --decimalgrouping none --ranges nospace
else
  echo "warning: SwiftFormat not installed, download from https://github.com/nicklockwood/SwiftFormat"
fi
```

I temporarily fixed with follow code

```bash
if which swiftformat >/dev/null; then
  swiftformat . --disable unusedArguments --commas inline --decimalgrouping none --ranges nospace
else
  echo "warning: SwiftFormat not installed, download from https://github.com/nicklockwood/SwiftFormat"
fi
```

Possible bug in this [commit](https://github.com/nicklockwood/SwiftFormat/commit/bde1992267a36d24f4754ebc95ac8ad814c61e63#diff-12d7cfe89917ea40fa2991c6606fb7a7R142)?

**Comment by nicklockwood** [comment]
@pampeano I believe this is now fixed in version 0.33.7, please verify. Thanks!

**Comment by pampeano** [comment]
Sorry for the delay! I tested with version 0.33.11 and its works! Thanks!
